addons/l10n_latam_check/__manifest__.py

```diff
@@ -14,6 +14,14 @@
 * allow to use different checkbooks type (deferred, electronic, current)
 * add an optional "payment date" for post-dated checks (deferred payments)
 * add a menu to track own checks
+* two modifications are done when using when using checkbooks:
```

Comment:
@jjscarafia just a minor thing, here there are 2 "when using", we should remove 1 repetition